fms_core/viewsets.py

```diff
@@ -2,12 +2,12 @@
 
 from collections import Counter
 from django.conf import settings
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, Group
 from django.db.models import Count, Q
 from django.http.response import HttpResponseNotFound, HttpResponseBadRequest
-from rest_framework import viewsets
+from rest_framework import viewsets, status
 from rest_framework.decorators import action
-from rest_framework.permissions import AllowAny
+from rest_framework.permissions import AllowAny, IsAuthenticated, DjangoModelPermissionsOrAnonReadOnly
```

Comment:
AnonReadOnly ? We need that ? Why would someone ask for permissions if they have them by default ?